values.yaml

```diff
@@ -413,6 +415,7 @@ mastodon:
     enabled: false
     exporterUri:
     namePrefix: mastodon
+    nameSeparator: /
```

Comment:
Why is this one different from the others above?

Also, if we set a default, shouldnt it be something that is k8s-compliant in the chart?

Reply:
Ah that's a good point...I made the default `/` because that's what it is in the code, but I agree, it doesn't make sense to keep it that when it's not good for k8s. I'll make it `-` by default.

Though I apologize, I'm not sure what you mean "why is it different"?

Reply:
On lines 188 and 388 you do not set this value

Reply:
Ohh I understand
Because these are the global values. lines 188 and 388 are deployment-specific, and will assume the global values if not set (`coalesce`).